src/shared/internal-calendar/day-cell/day-cell.style.tsx

```diff
@@ -0,0 +1,229 @@
+import styled, { css } from "styled-components";
+import { Color } from "../../../color";
+import { Text, TextStyleHelper } from "../../../text";
+import { CellType, LabelType } from "./types";
+
+// =============================================================================
+// STYLE INTERFACES
+// =============================================================================
+interface StyleProps {
+    $type?: CellType;
+    $shadow?: boolean;
+}
+
+interface LabelStyleProps {
+    $type: LabelType;
+    $disabled: boolean;
+    $interactive: boolean;
+}
+
+// =============================================================================
+// HELPERS
+// =============================================================================
+const getCellStyle = (props, type: CellType) => {
```

Comment:
Actually can just use the props and get `$type` from it? Were you trying to avoid dealing with the typings?

Reply:
oops this was a holdover from when the style props were distinct, the props arg can indeed access `$type` now